api_test.go

```diff
@@ -331,20 +333,14 @@ func runMultipleRegistrations(configFile string, count int) (int, chan string) {
 	return count, messages
 }
 
-func clearConfig() {
-	_ = os.Remove(deviceConfigPath)
-	_ = os.Remove(privConfigFilename(deviceConfigPath))
-}
-
-func removeTestConfig(configFile string) {
-	_ = os.Remove(configFile)
-	_ = os.Remove(privConfigFilename(configFile))
+func removeConfig() {
+	Fs.Remove(deviceConfigPath)
+	Fs.Remove(registeredConfigPath)
```

Comment:
There's no need for this if each test resets `Fs`. Remember that the filesystem is being faked in memory during tests.

Reply:
True